data.sql

```diff
@@ -4,3 +4,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8.00);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', '2020-02-08', 0, false, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', '2021-11-15', 2, true, 5.7);
```

Comment:
- [ ] Kindly change the `weight_kg` data to match with the provided, it's a negative value